10LessonTen/Checkpoint2/css/style.css

```diff
@@ -0,0 +1,130 @@
+h1{
+  display: block;
+  font-size: 2em;
+  margin-top: 0.67em;
+  margin-bottom: 0.67em;
+  font-weight: bold;
+}
+
+h3{
+  display: block;
+  font-size: 1.17em;
+  margin-top: 1em;
+  margin-bottom: 1em;
+  font-weight: bold;
+}
+
+h5{
+  display: block;
+  font-size: .83em;
+  margin-top: 1.67em;
+  margin-bottom: 1.67em;
+  font-weight: bold;
+}
+
+p{
+  display: block;
+  margin-top: 1em;
+  margin-bottom: 1em;
+}
+
+a{
+  text-decoration: none;
+  color: black;
+}
+
+nav>a{
+  display: inline-block;
+  margin-left: 20px;
+}
+
+main{
+  /*background-color: green;*/
+  width: 90vw;
+  margin-right: 5vw;
+  margin-left: 5vw;
+  padding-top: 15vh;
+}
+
+main>div:nth-child(1){
+  background-image: url(http://placehold.it/1280x500);
+  height: 500px;
+}
+
+main>div:nth-child(1)>div{
+  position: relative;
+  top: 25%;
+}
+
+main>div:nth-child(2)>*{
+  display: inline-block;
+  vertical-align: top;
+}
+
+main>div:nth-child(2)>div{
+  width: 50%;
+}
+
+main>div:nth-child(3)>.panel{
+  display: inline-block;
+  margin: 2%;
+  width: 20%;
+}
+
+main>div:nth-child(4)>*{
+  display: inline-block;
+  width: 41%;
+  padding: 2%;
+  margin: 2%;
+}
+
+main>div:nth-child(5){
+  height: 400px;
+  background-image: url(http://placehold.it/1280x400);
+}
+
+main>div:nth-child(5)>div{
+  position: relative;
+  top: 25%;
+}
+
+.panel{
+  /*background-color: blue;*/
+  margin-bottom: 75px;
+  text-align: center;
```

Comment:
make sure there is some padding left and right for your panel class here, otherwise content runs all the way to edge and it looks bad.